site/profile/manifests/android.pp

```diff
@@ -0,0 +1,33 @@
+
+class profile::android (
+        String $sdk_tools,
```

Comment:
Please correct the indentation. Please make sure that all new code conforms to Puppet style guide, I strongly suggest using http://puppet-lint.com/. @olegy-ait This may be useful for the rest of your team.